core/arch/arm/plat-mediatek/cros_huk.c

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023, The ChromiumOS Authors
```

Comment:
For legal reasons, we need a real name, see https://optee.readthedocs.io/en/latest/general/contribute.html#developer-certificate-of-origin